app/controllers/bananas2_controller.rb

```diff
@@ -66,13 +63,14 @@ def destroy
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_banana
-      @banana = Banana.find(params[:id])
-    end
 
-    # Never trust parameters from the scary internet, only allow the white list through.
-    def banana_params
-      params.require(:banana).permit(:name, :price)
-    end
+  # Use callbacks to share common setup or constraints between actions.
+  def set_banana
```

Comment:
__[reek]__
* [UnusedPrivateMethod] has the unused private instance method `set_banana` [:ref](https://github.com/troessner/reek/blob/master/docs/Unused-Private-Method.md)